output_derive/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+//! Custom derives for `output.rs`
+
+extern crate proc_macro;
+#[macro_use]
+extern crate quote;
+#[macro_use]
+extern crate syn;
+
+extern crate output;
+
+use self::proc_macro::TokenStream;
+use output::{human, json, Render};
```

Comment:
I think instead of doing this import here in the code that generates the code, you'll need to import it in the quoted part. I think there was some trick with making an `extern crate` work by putting the whole thing in a `const YOLO: () = { … }`, but maybe that's outdated by now.